app/src/main/java/com/capstone/weatherapp/WeatherCache.kt

```diff
@@ -0,0 +1,72 @@
+package com.capstone.weatherapp
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class CityCache(
+    @PrimaryKey val id: Int,
+    @ColumnInfo(name = "name") val name: String,
```

Comment:
Why do you specify the Column name here? Don't think you need to since you're specifying it to be "name" which is what the field is already named